sources/packages/web/src/components/common/restriction/AddRestriction.vue

```diff
@@ -1,46 +1,72 @@
 <template>
-  <modal-dialog-base :showDialog="showDialog" @dialogClosed="dialogClosed">
-    <template v-slot:content>
-      <v-container class="temporary-modal">
-        <formio
-          formName="addrestriction"
-          @loaded="formLoaded"
-          @changed="formChanged"
-          @submitted="submitRestriction"
-        ></formio>
-      </v-container>
-    </template>
-    <template v-slot:footer>
-      <check-permission-role :role="allowedRole">
-        <template #="{ notAllowed }">
-          <footer-buttons
-            primaryLabel="Add Restriction"
-            @primaryClick="addRestriction"
-            @secondaryClick="dialogClosed"
-            :disablePrimaryButton="notAllowed"
-        /></template>
-      </check-permission-role>
-    </template>
-  </modal-dialog-base>
+  <v-form ref="addRestrictionForm">
+    <!-- TODO min-width has to be confirmed  -->
+    <modal-dialog-base
+      title="Add new restriction"
+      :showDialog="showDialog"
+      max-width="730"
+      min-width="730"
```

Comment:
Was that the only modal that required a `min-width` or all the other ones are missing it?
Which confirmation is pending to be confirmed and have the TODO removed?

Reply:
No there are 2 modals, ViewRestriction and this, when a modal do not have a title to it. the modal becomes crumbled. So that was the reason to it. We can add it in the overall component level. Devs can discuss.